Corona-Warn-App/src/main/assets/privacy_de.html

```diff
@@ -1716,6 +1764,16 @@ <h3>
     Bestätigung der Echtheit Ihrer App findet unmittelbar statt, nachdem Sie Ihr Einverständnis
     erteilt haben.
 </p>
+<h3>
+    i. Einverständnis „Aktualisierungsfunktion für COVID-Zertifikate“
+</h3>
+<p>
+    Da die Aktualisierung eines digitalen COVID-Zertifikats nach der Erteilung Ihres
+    Einverständnisses sofort durchgeführt wird, kann das Einverständnis nicht mehr zurückgenommen
+    werden. Wenn die Aktualisierung nicht erfolgreich war oder Sie ein weiteres digitales
+    COVID-Zertifikat aktualisieren möchten, müssen Sie für jeden einzelnen Aktualisierungsvorgang
+    ein gesondertes Einverständnis erteilen.
```

Comment:
Just as a technical remark:
It depends on _when_ in the process Ubirch stores the hash:
If they store it right at the beginning of the reissuance process, and there is a loss of internet connection before the new certificate can be sent to the client, then the user won't be able to trigger the reissuance again (fyi @mlenkeit ) as the already stored hash then will prevent a new reissuance (as an anti-measure for (D)DoSing).
If this is the case, UX/UI should have some words for clarification (e.g., "Due to an error, your DCC can't be reissued. Please see your pharmacist for further support.) It would be interesting, whether in pharmacies then reissuing would also be blocked due to the stored hash...

If Ubirch stores the hash _after_ the **successful** submission to the client, then DDoSing with spoofed sender IPs would still be possible.

If the 365-days period is rather to avoid abuse, then it could be good to discuss to shorten that period, e.g. to 24 hours, what should be enough to prevent (D)DoSing.
Unless Ubirch or other stakeholders don't charge the BMI for any re-issued certificate (per certificate), I don't see any downside to lower the storage period to 24 hours.
Probably the most common/likely case for re-issuance will be, when the DCC is experied from the period of _technical_ validity (COSE header). DCCs with expired technical validity could _always_ be reissued.

cc: @thinkberg